pkg/apis/resource/install/install.go

```diff
@@ -36,5 +36,5 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(resource.AddToScheme(scheme))
 	utilruntime.Must(v1alpha3.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1alpha3.SchemeGroupVersion))
+	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1alpha3.SchemeGroupVersion))
```

Comment:
I added a comment about "Downgrades from 1.32 to 1.31 are not supported" to the release note.

I am not sure whether that warrants an `ACTION REQUIRED: when downgrading from 1.32 to 1.31, delete ..." ?

Reply:
In order to support serialized data, I thought we don't actually REMOVE old APIs.  But it does look like we remove old ALPHA APIs - but only sometimes.  e.g. discovery/v1a1 was removed but admissionregistration/v1alpha1 still has VAP.

I am not sure this part is OK.  If someone uses the beta, and then rolls back, we should not penalize them.

Reply:
They roll back to an alpha where we can't fix some knows bugs because our cherry-pick guidelines treat fixes for off-by-default alpha features as [not eligible](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md#what-kind-of-prs-are-good-for-cherry-picks). I think admins who need to roll back shouldn't expect the feature to work and are better of disabling workloads.